CODEOWNERS

```diff
@@ -1,20 +1,34 @@
-*           @amanda-tarafa
+# There are two aspects that should be taken into account when reviewing samples on this repo.
+#   * That the samples are meaningful and consistent with similar samples in other languages
+#     and with what is stated in documentation. Some sample sets have specific owners that
+#     are in charge of this aspect.
+#   * The idiomacy of the sample and the consistency with the rest of the samples on this repo.

```

Comment:
I think this should be "idiomaticity" - but it could be clearer as "the extent to which the sample is idiomatic C# and consistent with the rest of the samples in this repo"

Reply:
Yes, will change.